lighthouse/lighthouserc.js

```diff
@@ -1,17 +1,19 @@
 const AUDIT_URLS = {
   FRONT_PAGE: {
-    live: ['https://www.bbc.com/arabic'],
-    test: ['https://www.test.bbc.com/arabic'],
+    live: ['https://www.bbc.com/serbian/cyr'],
+    test: ['https://www.test.bbc.com/serbian/cyr?renderer_env=live'],
   },
 
   IDX: {
     live: ['https://www.bbc.com/persian/afghanistan'],
-    test: ['https://www.test.bbc.com/persian/afghanistan'],
+    test: ['https://www.test.bbc.com/persian/afghanistan?renderer_env=live'],
```

Comment:
Using `renderer_env` to display live data (instead of using test data which can be very out of date)